src/controllers/userController.js

```diff
@@ -2,7 +2,7 @@ import { randomBytes } from 'crypto';
 import users from '../db/users';
 import parcels from '../db/parcel';
 
-export default {
+class Reflection {
```

Comment:
Why are you naming all your classes `Reflection`? 🤔

Reply:
That's a clear mistake, thought I could give my class a different name. I'll fix it now